.github/workflows/ci.yml

```diff
@@ -19,6 +19,12 @@ jobs:
           sudo gem install --no-document fpm
           sudo apt-get install -y rpm
 
-      - name: run test
+      - name: run apisix packaging test
         run: |
-          make package type=rpm app=apisix version=2.3 code_tag=2.3
+          make package type=rpm app=apisix version=2.3 checkout=2.3
+          make package type=rpm app=apisix version=2.2 checkout=v2.2
+          make package type=rpm app=apisix version=master checkout=master
+
+      - name: run apisix-dashboard packaging test
```

Comment:
we can run it in a new CI

`run apisix packaging test` and `run apisix-dashboard packaging test`, Run them in parallel.

we can do this after this PR merged.

Reply:
@idbeta

Reply:
In this way, we can complete the ci in 11 minutes 

![image](https://user-images.githubusercontent.com/6814606/109141412-aca28b00-7798-11eb-86bf-94ca2598ca40.png)

Reply:
ok, i got it.